lib/facter/nginx_version.rb

```diff
@@ -2,7 +2,7 @@
   setcode do
     if Facter.value('kernel') != 'windows' && Facter::Util::Resolution.which('nginx')
       nginx_version = Facter::Util::Resolution.exec('nginx -v 2>&1')
-      %r{nginx version: nginx\/([\w\.]+)}.match(nginx_version)[1]
+      %r{nginx version: (nginx|openresty)\/([\w\.]+)}.match(nginx_version)[1]
```

Comment:
Yep, adding an extra capture zone means you have to change this to the second selector as Alex says.

The tests are failing because of this:

```
 1) Facter::Util::Fact with current version output format should eq "1.8.1"
     Failure/Error: it { expect(Facter.fact(:nginx_version).value).to eq('1.8.1') }
     
       expected: "1.8.1"
            got: "nginx"
     
       (compared using ==)
     # ./spec/unit/facter/util/fact_nginx_version_spec.rb:11:in `block (3 levels) in <top (required)>'
```